### PR TITLE
Add Sinatra/Sequel example to README

### PR DESCRIPTION
**Pull request by mwpastore**
I specifically want to call out Sequel's `:tactical_eager_loading` plugin, which makes a huge difference in the number of queries fired when sideloading related records, but thought it might be nice to add a full Sinatra example. Let me know your thoughts. Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -540,6 +541,104 @@ ActionDispatch::Http::Parameters::DEFAULT_PARSERS[Mime::Type.lookup(JSONAPI::MIM
...
+Sequel::Model instances.
+
+```ruby
+require 'sequel'
+require 'sinatra/base'
+require 'json/ext'
+require 'jsonapi-serializers'
+
+class Post < Sequel::Model
+  plugin :tactical_eager_loading # N.B.
```
Comment (fotinakis) [inline README.md]: Can you drop this `# N.B.` and the one below? I think it's kind of an obscure reference most people won't get.

File: README.md [hunk 2]
```diff
@@ -540,6 +541,104 @@ ActionDispatch::Http::Parameters::DEFAULT_PARSERS[Mime::Type.lookup(JSONAPI::MIM
...
+        options[:is_collection] = true
+        JSONAPI::Serializer.serialize(models, options)
+      end
+    end
+
+    before do
+      @data = parse_request_body if request.body.size > 0
+    end
+
+    get '/posts', :provides => :jsonapi do
```
Comment (fotinakis) [inline README.md]: For style consistency, can you make this `provides: :jsonapi` and same below?

Comment (fotinakis) [inline README.md]: Same with `symbolize_names: true` above.

**Comment by fotinakis** [comment]
I'm good with this, just a couple of style comments above. Thanks!

File: README.md (hunk 1 above)
Comment (mwpastore) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (mwpastore) [inline README.md]: Good catches. Done.

**Comment by mwpastore** [comment]
Great! Sorry for so many force-pushes; I should have read all your comments at the same time instead of reading and acting on them... serially. *groan* Let me know if you'd like to see any other changes.

**Pull request merged**